### PR TITLE
Initial update of Charter

### PR DESCRIPTION
**Pull request by tbrooke**

### CODE REVIEW

File: accord-project-technical-charter.md [hunk 1]
```diff
@@ -10,92 +10,49 @@ a. The mission of the Project is to develop open source software tools for legal
...
-ii. The Chair of the the Technical Steering Committee.
+The Project shall be governed by the Technical Steering Committee (TSC) and the TSC shall be responsible for:
 
-b. Conduct of Meetings
+    (a. approving a budget directing the use of any funds raised by, or used by, AP;
 
-i. Governing Board meetings shall be limited to the Governing Board representatives and follow the requirements for quorum and voting established by the Governing Board.
+    b. electing a Chair of AP to preside over TSC meetings, authorize expenditures approved by the budget and manage any necessary day-to- day operations;
+     
+    c. electing a treasurer with authority for signing checks and responsiblity for credit card purchases;
```
Comment (mttrbrts) [inline accord-project-technical-charter.md]: Can we generalise this? The main function will be approving expenses through the Linux Foundation. We don't hold a check book, or a credit card.

File: accord-project-technical-charter.md [hunk 2]
```diff
@@ -10,92 +10,49 @@ a. The mission of the Project is to develop open source software tools for legal
...
+    e. administering, provisioning, and managing AP involvement in external projects; 
 
-The Governing Board shall be responsible for:
+    f. adopting and maintaining policies or rules and procedures for AP (subject to LF approval); 
 
-i. approving a budget directing the use of any funds raised by, or used by, AP;
+    g. voting on all decisions or matters coming before the TSC.
 
-ii. electing a Chair of AP to preside over Governing Board meetings, authorize expenditures approved by the budget and manage any necessary day-to- day operations;
+    h. TSC voting members are initially the existing TSC members and the former Governing Board members as of December ___, 2023 with existing TSC members retaining their present term and existing Governing Board Members starting a new term in December, 2023. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
```
Comment (mttrbrts) [inline accord-project-technical-charter.md]: Shall we aim for 14th December, i.e. 2 weeks after tomorrow's TSC meeting?

File: accord-project-technical-charter.md [hunk 3]
```diff
@@ -10,92 +10,49 @@ a. The mission of the Project is to develop open source software tools for legal
...
+a. While the Project aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Project forward, the voting members of the TSC will vote on a one vote per voting member basis.
 
-vii. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+b. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
 
-viii. coordinating any marketing, events, or communications regarding the Project.
+c. decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
 
-## 4. TSC Voting
+d. In the event a vote cannot be resolved by the TSC, any voting member of the TSC may refer the matter to the Series Manager for assistance in reaching a resolution.
```
Comment (mttrbrts) [inline accord-project-technical-charter.md:44]: What is the Series Manager?

File: accord-project-technical-charter.md [hunk 4]
```diff
@@ -10,92 +10,49 @@ a. The mission of the Project is to develop open source software tools for legal
...
-
-ii. approving, modifying and disbanding Working Groups and sub-projects;
-
-iii. creating sub-committees to focus on cross-project issues and requirements;
-
-iv. appointing representatives to work with other open source or open standards communities;
-
-v. establishing meeting procedures, community norms, workflows, issuing releases, and security issue reporting policies;
-
-vi. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the “Series Manager”) to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
```
Comment (mttrbrts) [inline accord-project-technical-charter.md:82]: This section looks helpful to retain, perhaps?

File: accord-project-technical-charter.md [hunk 5]
```diff
@@ -10,92 +10,49 @@ a. The mission of the Project is to develop open source software tools for legal
...
 
-i. Two (2) directors of the Project; and 
-ii. The Chair of the the Technical Steering Committee.
+The Project shall be governed by the Technical Steering Committee (TSC) and the TSC shall be responsible for:
 
-b. Conduct of Meetings
+    (a. approving a budget directing the use of any funds raised by, or used by, AP;
 
-i. Governing Board meetings shall be limited to the Governing Board representatives and follow the requirements for quorum and voting established by the Governing Board.
+    b. electing a Chair of AP to preside over TSC meetings, authorize expenditures approved by the budget and manage any necessary day-to- day operations;
```
Comment (mttrbrts) [inline accord-project-technical-charter.md]:
```suggestion
    b. electing a Chair of AP to preside over TSC meetings, authorize expenditures approved by the budget and manage any necessary day-to-day operations;
```

File: accord-project-technical-charter.md (hunk 1 above)
Comment (tbrooke) [reply accord-project-technical-charter.md]: If they hold the credit cards, check book we may not need a treasuurer - Why not just take it out?

File: accord-project-technical-charter.md (hunk 3 above)
Comment (tbrooke) [reply accord-project-technical-charter.md:44]: Good Question the the Project is a Series LLC - which is a hybrid LLC -- An LLC has managers - But I am thinking that LSF may have a manager over the Series LLCs and that is what it is referring to -- I will check with them

File: accord-project-technical-charter.md [hunk 6]
```diff
@@ -10,92 +10,47 @@ a. The mission of the Project is to develop open source software tools for legal
...
+    e. adopting and maintaining policies or rules and procedures for AP (subject to LF approval); 
 
-i. approving a budget directing the use of any funds raised by, or used by, AP;
+    f. voting on all decisions or matters coming before the TSC.
 
-ii. electing a Chair of AP to preside over Governing Board meetings, authorize expenditures approved by the budget and manage any necessary day-to- day operations;
+    g. TSC voting members are initially the existing TSC members and the former Governing Board members as of December ___, 2023 with existing TSC members retaining their present term and existing Governing Board Members starting a new term in December, 2023. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
 
-iii. overseeing all Project business and marketing matters;
+    h. TSC Members shall have a two (2) year term and they may run for consequtive terms.
```
Comment (DianaLease) [inline accord-project-technical-charter.md]:
```suggestion
    h. TSC Members shall have a two (2) year term and they may run for consecutive terms.
```

File: accord-project-technical-charter.md [hunk 7]
```diff
@@ -10,92 +10,47 @@ a. The mission of the Project is to develop open source software tools for legal
...
 
-iv. administering, provisioning, and managing AP involvement in external projects; 
+    i. the TSC may (1) establish work flow procedures for the submission, approval, and closure/archiving of projects, (2) set requirements for the promotion of Contributors to Maintainer status, as applicable, and (3) amend, adjust, refine and/or eliminate the roles of Contributors, Maintiners and Committers, and create new roles, and publicly document any TSC roles, as it sees fit.
```
Comment (DianaLease) [inline accord-project-technical-charter.md]:
```suggestion
    i. the TSC may (1) establish work flow procedures for the submission, approval, and closure/archiving of projects, (2) set requirements for the promotion of Contributors to Maintainer status, as applicable, and (3) amend, adjust, refine and/or eliminate the roles of Contributors, Maintainers and Committers, and create new roles, and publicly document any TSC roles, as it sees fit.
```

File: accord-project-technical-charter.md [hunk 8]
```diff
@@ -10,92 +10,47 @@ a. The mission of the Project is to develop open source software tools for legal
...
+d. In the event a vote cannot be resolved by the TSC, any voting member of the TSC may refer the matter to the Series Manager for assistance in reaching a resolution.
 
-a. While the Project aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Project forward, the voting members of the TSC will vote on a one vote per voting member basis.
+## 5. TSC Elections
 
-b. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
+a. TSC members shall be elected after open nominations adopted by the procedures set forth in the policies of the projects.
 
-c. Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
+b. Any individual that have made at least one commit to any Accord Project repository is entitled to vote in the elction.
```
Comment (DianaLease) [inline accord-project-technical-charter.md]:
```suggestion
b. Any individual that have made at least one commit to any Accord Project repository is entitled to vote in the election.
```

**Review by DianaLease — COMMENTED** [review]
Added suggestions to fix just a few small typos

**Review by DianaLease — APPROVED** [review]

File: accord-project-technical-charter.md [hunk 9]
```diff
@@ -10,92 +10,47 @@ a. The mission of the Project is to develop open source software tools for legal
...
+    d. administering, provisioning, and managing AP involvement in external projects; 
 
-The Governing Board shall be responsible for:
+    e. adopting and maintaining policies or rules and procedures for AP (subject to LF approval); 
 
-i. approving a budget directing the use of any funds raised by, or used by, AP;
+    f. voting on all decisions or matters coming before the TSC.
 
-ii. electing a Chair of AP to preside over Governing Board meetings, authorize expenditures approved by the budget and manage any necessary day-to- day operations;
+    g. TSC voting members are initially the existing TSC members and the former Governing Board members as of December ___, 2023 with existing TSC members retaining their present term and existing Governing Board Members starting a new term in December, 2023. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
```
Comment (mttrbrts) [inline accord-project-technical-charter.md]:
```suggestion
    g. TSC voting members are initially the existing TSC members and the former Governing Board members as of January 31, 2023 with existing TSC members retaining their present term and existing Governing Board Members starting a new term in December, 2023. Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person.
```

**Review by mttrbrts — APPROVED** [review]

**Review by dselman — APPROVED** [review]

**Pull request merged**